clang_delta/tests/test_clang_delta.py

```diff
@@ -588,105 +630,216 @@ def test_remove_unused_function_class(self):
             'remove-unused-function/class.cc',
             '--transformation=remove-unused-function --counter=1',
         )
```

Comment:
Ditto with old test assertions for `remove-unused-function` - we can delete them once we switch everything to hints, which is why I didn't invest into any clever way of parameterizing the tests for "with hints and w/o hints".